src/enzo/ActiveParticleFinalize.C

```diff
@@ -0,0 +1,94 @@
+/***********************************************************************
+/
+/  ACTIVE PARTICLE INITIALIZATION
+/
+/  written by: John Wise
+/  date:       March, 2009
+/  modified1:  November, 2011 (JHW) -- converting to active particles
+/
+/  PURPOSE: Contains all routines to finalize the star particles.
+/
```

Comment:
Update the text for purpose of this file.

Reply:
Done